server/db/models/index.js

```diff
@@ -1,23 +1,27 @@
 const User = require('./user')
-// const ListItem = require('./list-item')
+const ListItem = require('./list-item')
 const Trip = require('./trip')
-// const Rating = require('./rating')
+const Rating = require('./rating')
 
 User.belongsToMany(Trip, {as: 'subscriber', through: 'trip_subscribers'})
 Trip.belongsToMany(User, {as: 'subscriber', through: 'trip_subscribers'})
```

Comment:
Suggest following camel case for TripSubscribers so potential requests to the join table are similar to those of other models

Reply:
Camel case makes it more difficult to use with psql. That's why I changed it. Is that ok?

Reply:
Gotcha, keep it as is!